src/meta/greedy_load_balancer.cpp

```diff
@@ -1031,7 +1069,7 @@ bool greedy_load_balancer::get_cluster_migration_info(const meta_view *global_vi
     for (const auto &kv : all_apps) {
         const std::shared_ptr<app_state> &app = kv.second;
         if (is_ignored_app(app->app_id) || app->is_bulk_loading || app->splitting()) {
-            return false;
+            continue;
```

Comment:
Why the cluster level load balance should not triggered if a app is ignored? I think ignoring this app is enough

Reply:
If an app is ignored, it is possible for the cluster balance not finished, how to ignore one app and suppose the whole cluster balanced?

Reply:
But it's unreasonable to diable load balance if an app is ignored. 
And the app ignoring means that the cluster is balanced ignoring this ignored app.

Reply:
Yes, actually I agree with your opinion. In my view, if there is an app is ignored, we should use another balance strategy, as current implementation, app level load balance, not the cluster level load balance. Because in cluster level, it is not balanced, users may be confused that why the cluster is not balanced when they choose cluster-level balance strategy. 

If you insist on your implementation, I suggest adding some comments or other things to notify that some apps are ignored in balance round.

Reply:
I added a log for it.

Reply:
actually, I think `ignored app` case is different from `bulkload` and `split`, `balance` should break but loop to wait the `split` or `bulkload` completed and continue to balance, but `ignored app` just can break and no need care it.

or if some app is `split` and `bulkload`, the balance shouldn't be started

Reply:
I have talked about it with @hycdong , and we think we should ignore the app which is doing bulk load. And when the bulk load is done, it will be balanced in the later.